### PR TITLE
Tutorials drafts

### PR DESCRIPTION
**Pull request by fangohr**
- First installation tutorial created
- outlined index notebook: this is not fully developed yet. I am hoping we can have an tutorial index notebook that 
  points to all available notebooks, so that one can navigate those on Binder. Or maybe this is causing more complications than it is useful.

### CODE REVIEW

**Comment by fangohr** [comment]
Would be nice to bring this online soon, so we can experiment with links to binder etc.

File: tools/pelican/content/ipynb/tutorial-installation.ipynb [hunk 1]
```diff
@@ -0,0 +1,12866 @@
...
+    "What technically happens is that (through the [binder project](http://mybinder.org)) a container (a bit like a small virtual machine) is created just for you at that point on a server somewhere on the planet. You can use this container from your browser, but any commands you execute are actually executed on this server somewhere else (i.e. \"in the cloud\"). You can use this container as long as you like, but there is a time-out: after 10 minutes of inactivity, your session will be closed.\n",
+    "\n",
+    "This way of using OSCOVIDA in the cloud is very convenient, because you don't need to install any software on your own computer/tablet etc.\n",
+    "\n",
+    "However, there are a number of limitations to be aware:\n",
+    "\n",
+    "- any interactive session you have, will be deleted after 10 minutes of inactivity\n",
+    "- at that point, your notebooks is also lost\n",
+    "- if you want to keep the notebook, you can go to `File -> Download as -> Notebook (.ipynb)` before your session expires\n",
+    "- if you have have saved such a notebook, you can upload it and then continue to work on it in your next session."
```
Comment (RobertRosca) [inline tools/pelican/content/ipynb/tutorial-installation.ipynb:33]: Might be worth noting that some of the binder services may block downloads for security reasons, at least until we find a solution for that issue.

File: tools/pelican/content/ipynb/tutorial-installation.ipynb [hunk 2]
```diff
@@ -0,0 +1,12866 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Alternatively, you can install OSCOVIDA on your own computer, and then all the files you create and modify are saved locally on your computer.\n",
+    "\n",
+    "OSCOVIDA is a python package, and makes use of other Python libraries such as `matplotlib` for creation of plots and `numpy` and `pandas` for data handling.\n",
+    "\n",
+    "If you know Python and have Python (version 3.6 or newer) installed already, then you can install `osvodia` using `pip install oscovida`.\n",
```
Comment (RobertRosca) [inline tools/pelican/content/ipynb/tutorial-installation.ipynb:51]:
```suggestion
    "If you know Python and have Python (version 3.6 or newer) installed already, then you can install `oscovida` using `pip install oscovida`.\n",
```

File: tools/pelican/content/ipynb/tutorial-installation.ipynb [hunk 3]
```diff
@@ -0,0 +1,12866 @@
...
+       "     <use style=\"fill:#348abd;fill-opacity:0.3;stroke:#348abd;stroke-opacity:0.3;\" x=\"277.142597\" xlink:href=\"#ma0c9089e31\" y=\"498.873586\"/>\n",
+       "     <use style=\"fill:#348abd;fill-opacity:0.3;stroke:#348abd;stroke-opacity:0.3;\" x=\"280.235724\" xlink:href=\"#ma0c9089e31\" y=\"500.151057\"/>\n",
+       "     <use style=\"fill:#348abd;fill-opacity:0.3;stroke:#348abd;stroke-opacity:0.3;\" x=\"283.32885\" xlink:href=\"#ma0c9089e31\" y=\"502.409008\"/>\n",
+       "     <use style=\"fill:#348abd;fill-opacity:0.3;stroke:#348abd;stroke-opacity:0.3;\" x=\"286.421976\" xlink:href=\"#ma0c9089e31\" y=\"502.290552\"/>\n",
+       "     <use style=\"fill:#348abd;fill-opacity:0.3;stroke:#348abd;stroke-opacity:0.3;\" x=\"289.515103\" xlink:href=\"#ma0c9089e31\" y=\"502.035503\"/>\n",
+       "     <use style=\"fill:#348abd;fill-opacity:0.3;stroke:#348abd;stroke-opacity:0.3;\" x=\"292.608229\" xlink:href=\"#ma0c9089e31\" y=\"503.997437\"/>\n",
+       "     <use style=\"fill:#348abd;fill-opacity:0.3;stroke:#348abd;stroke-opacity:0.3;\" x=\"295.701356\" xlink:hre
```
Comment (RobertRosca) [inline tools/pelican/content/ipynb/tutorial-installation.ipynb:12841]:
```suggestion
    "If you can see the plot, you have successfully install `oscovida`, and are ready to use it."
```

Comment (RobertRosca) [reply tools/pelican/content/ipynb/tutorial-installation.ipynb:12841]: Er... not sure if the suggestion is selecting the wrong line in the preview for you too, the suggestion should be to fix the typo `sucessfully` -> `successfully`.

Comment (fangohr) [reply tools/pelican/content/ipynb/tutorial-installation.ipynb]: I have fixed that locally; thanks!

**Pull request merged**